lib/slash-commands/ext/ext.js

```diff
@@ -41,31 +52,96 @@ class Ext {
             callback = arguments[1];
         }
 
-        Util.readJsonFromFile(FILE, (err, data) => {
-            if (err) {
-                throw err;
+        if (this._attemptedToReadSpreadsheetConfigOnce) {//if we already tried to load the config file on a previous request...
```

Comment:
I've moved some of the logic that deals with presence/absence of the spreadsheet config file to a new method called `_readConfig`, . And for semantics purposes i've changed the name of the method `find` to `getExtensions`.